bin/validateSchema.ts

```diff
@@ -201,8 +193,9 @@ export async function validateVisitsResponseSchema(testSubscriptions: TestSubscr
  * Validates Webhook schema
  */
 async function validateWebhookSchema() {
-  console.log('\nValidating Webhook schema: \n');
-  const webhookSchema = convertOpenApiToJsonSchema(OPEN_API_SCHEMA, '#/definitions/Webhook');
+  const schemaName = 'Webhook';
+  console.log(`\n⚡ ${getCurrentFunctionName()}: Validating ${schemaName} schema: \n`);
```

Comment:
nit: since this same `console.log` message format is used in multiple places, we could extract a separate function from it.

Reply:
Done in: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/200/commits/42bd7aa3162c19e170fd0c9b4ec353c75e93ea1f